pkg/regressionallowances/regressions_4.17.json

```diff
@@ -0,0 +1,52 @@
+[
+  {
+    "JiraComponent": "Management Console",
+    "TestID": "openshift-tests:f9552d2b4b817d19bdcd666a7b8d1ece",
+    "TestName": "[bz-Management Console] clusteroperator/console should not change condition/Available",
+    "JiraBug": "https://issues.redhat.com/browse/OCPBUGS-39580",
+    "ReasonToAllowInsteadOfFix": "Test flaked in 4.16, became more strict in 4.17",
+    "variant": {
+      "variants": {
+        "Network": "ovn",
+        "Upgrade": "none",
+        "Architecture": "amd64",
+        "Platform": "vsphere",
+        "FeatureSet": "default",
+        "Suite": "serial",
+        "Topology": "ha",
+        "Installer": "upi"
+      }
+    },
+    "PreviousSuccesses": 66,
+    "PreviousFailures": 0,
+    "PreviousFlakes": 1,
+    "RegressedSuccesses": 27,
+    "RegressedFailures": 4,
+    "RegressedFlakes": 0
+  },
+  {
+    "JiraComponent": "kube-apiserver",
+    "TestID": "openshift-tests:d6b41cee7afca1c2a0b52f9e6975425f",
+    "TestName": "[bz-kube-apiserver][invariant] alert/KubeAPIErrorBudgetBurn should not be at or above info",
+    "JiraBug": "https://issues.redhat.com/browse/OCPBUGS-42083",
+    "ReasonToAllowInsteadOfFix": "TBD Reason",
```

Comment:
Since I filed this bug I just want to chime with my opinion:

This bug was found during the final stabilization period of our release.  We believe this is a real problem and must be fixed, but investigation suggested it was not severe enough to change our release date.